controllers/operator/manager.go

```diff
@@ -191,8 +193,8 @@ func (m *ODLMOperator) ListOperandRequestsByConfig(ctx context.Context, key type
 	return
 }
 
-// GetSubscription gets Subscription from a name
-func (m *ODLMOperator) GetSubscription(ctx context.Context, name, namespace string, packageName ...string) (*olmv1alpha1.Subscription, error) {
+// GetSubscription gets Subscription by name and package name
+func (m *ODLMOperator) GetSubscription(ctx context.Context, name, namespace string, packageName string) (*olmv1alpha1.Subscription, error) {
```

Comment:
Could you merge all three `string` parameters to `(name, namespace, packageName string)`?

Reply:
Nice catch